package/dist/etc/hyper/config

```diff
@@ -1,21 +1,45 @@
-# Boot kernel for KVM/XEN
+# configurations for hyperd
+
+# Root directory for hyperd
+# Root=/var/lib/hyper/
+
+# Specify the hypervisor: libvirt, qemu, qemu-kvm, kvm, xen (for linux)
+#                         vbox (for mac).
```

Comment:
VirtualBox should work on linux, it does support all platforms.  The hyper may use that when virtualbox is available on any system.